server/JsDbg.Core/IDebugger.cs

```diff
@@ -112,7 +112,7 @@ public interface IDebugger {
         Task<IEnumerable<SConstantResult>> LookupConstants(string module, string type, ulong constantValue);
         Task<SConstantResult> LookupConstant(string module, string type, string constantName);
         Task<SFieldResult> LookupField(string module, string typename, string fieldName);
-        Task<SSymbolResult> LookupGlobalSymbol(string module, string symbol, string typeName);
+        Task<SSymbolResult> LookupGlobalSymbol(string module, string symbol, string typeName, string nameSpace);
```

Comment:
nit: name**s**pace, to be consistent with the client

Reply:
Unfortunately namespace is a reserved identifier and I can't use it as a parameter name :(

Reply:
Oh yeah, sorry I missed that.